### PR TITLE
Mobile Breakpoint identification Resolved

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#43245](https://central.tri.be/issues/43245)

### CODE REVIEW

File: src/resources/js/tribe-events.js [hunk 1]
```diff
@@ -684,10 +684,11 @@ Date.prototype.format = function( mask, utc ) {
 		mobile_class          : function() {
 			var $body = $( 'body' );
 
+			console.log( tribe_ev.data.v_width, tribe_ev.data.mobile_break );
```
Comment (borkweb) [inline src/resources/js/tribe-events.js]: You left some debug output in here.

File: src/resources/js/tribe-events.js [hunk 2]
```diff
@@ -1192,14 +1193,27 @@ Date.prototype.format = function( mask, utc ) {
...
+		if ( $tribe_events.length ) {
+			var $breakpoint_holder = $tribe_events.tribe_has_attr( 'data-mobilebreak' );
+
+			if ( false === $breakpoint_holder ) {
+				$breakpoint_holder = $tribe_events.find( '[data-mobilebreak]' ).eq( 0 );
+			} else {
+				$breakpoint_holder = $tribe_events;
+			}
+
+			td.mobile_break = parseInt( $breakpoint_holder.data( 'mobilebreak' ) );
```
Comment (borkweb) [inline src/resources/js/tribe-events.js]: You should provide the radix when using `parseInt`:

```js
parseInt( blahblah, 10 );
```

Comment (bordoni) [inline src/resources/js/tribe-events.js]: Ohh yeah I gotta reactivite my JSHint

File: src/resources/js/tribe-events.js (hunk 1 above)
Comment (bordoni) [inline src/resources/js/tribe-events.js]: Thxxx

**Comment by bordoni** [comment]
Ready for CR again.

**Comment by borkweb** [comment]
:ocean:

**Pull request merged**